account_fiscal_year/models/date_range.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Author: Damien Crier
+# Copyright 2016 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import models
+from openerp.tools import DEFAULT_SERVER_DATE_FORMAT
+
+
+class DateRange(models.Model):
+    _inherit = 'date.range'
+
+    def find_daterange_fy_start(self, date):
```

Comment:
@sbidoul used in another module there: https://github.com/oca-sorrento/account-financial-reporting/blob/push-here/account_financial_report_qweb/wizard/ledger_report_wizard.py#L150